docs/index.md

```diff
@@ -59,10 +36,15 @@ communication.
 
 For more information and resources, please visit the [Getting Help project page](https://docs.docker.com/project/get-help/).
 
-## Registry documentation
+## Documentation
+
+Basics:
+
+ - [Deployment](deploying.md)
+ - [Configuration](configuration.md)
+
+Advanced:
 
- - [Deploying a registry](deploying.md)
- - [Configure a registry](configuration.md)
  - [Storage driver model](storagedrivers.md)
  - [Working with notifications](notifications.md)
- - [Registry API v2](spec/api.md)
+ - [Registry API](spec/api.md)
```

Comment:
In general, source-files should end with a newline. If a new line (text) is added at the end, it's only a single LOC, if the newline is missing, it will be two LOC's.

It's a nit but also convention (hence GitHub marking it with a red "missing newline" icon)